src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
+
+  if (days < 3) {
+    return days * costPerDay;
+  }
+
+  if (days < 7) {
+    return days * costPerDay - threeDayDiscount;
+  } else {
+    return days * costPerDay - sevenDayDiscount;
+  }
```

Comment:
`return` ends function execution, so else here is redundant.
```suggestion
  if (days < 7) {
    return days * costPerDay - threeDayDiscount;
  } 
  
  return days * costPerDay - sevenDayDiscount;
```